install/install.js

```diff
@@ -1,3 +1,6 @@
 const install = require('../src/install');
+const exec = require('@actions/exec')
 
+
+exec.exec('echo "TELEPRESENCE_GITHUB_ACTION_INTEGRATION=true" >> $GITHUB_ENV')
```

Comment:
also looks like the setting of the env var is done within the `telepresenceInstall` function too.